source/settings.c

```diff
@@ -91,6 +98,11 @@ static void ani_callback(int sel)
 	apollo_config.doAni = !sel;
 }
 
+static void btn_fade_callback(int sel)
```

Comment:
let's remove it and use the animation setting

Reply:
Fixed